model/crystal.h

```diff
@@ -1028,6 +1049,8 @@ namespace dxtbx { namespace model {
   protected:
     cctbx::sgtbx::space_group space_group_;
     cctbx::uctbx::unit_cell unit_cell_;
+    cctbx::uctbx::unit_cell postrefined_cell_;
+    bool postrefined_cell_is_set_ = false;
```

Comment:
`has_renamed_cell` would be more conventional

Reply:
i.e. where `renamed` is whatever we consider a replacement to `postrefined` to be